helm_charts/ingress/cert-manager.yaml

```diff
@@ -0,0 +1,6754 @@
+# Copyright YEAR The Jetstack cert-manager contributors.
```

Comment:
Why are you adding the cert-manager file here? Is the helm chart not sufficient?

Reply:
hi, I took an example for the ingress setup for airflow, and on the clean AKS cluster, the ingress rule failed about missing cert-manager. I agree that this should be configured in ingress helm chat. I will check the options here.

Reply:
Good catch!